pkg/devspace/kubectl/client.go

```diff
@@ -250,10 +285,13 @@ func (client *client) PrintWarning(generatedConfig *generated.Config, noWarning,
 	}
 
 	// Info messages
-	log.Infof("Using namespace '%s'", ansi.Color(client.namespace, "white+b"))
-	log.Infof("Using kube context '%s'", ansi.Color(client.currentContext, "white+b"))
+	log.Infof("Using namespace '%s'", ansi.Color(namespace, "white+b"))
+	log.Infof("Using kube context '%s'", ansi.Color(kubeContext, "white+b"))
 
-	return nil
+	if resetClient {
+		return NewClientFromContext(kubeContext, namespace, false, client.kubeLoader)
```

Comment:
I think we should actually switch context here if the user decides to switch

Reply:
done.

Reply:
We're only reseting the client if user selects different context or namespace when asked.

Reply:
@pratikjagrut yes, but the problem would be that the user would get asked this question everytime after the question, e.g.:
```
devspace dev -n test
devspace use namespace test2
devspace dev # <- question here, I select test
devspace dev # <- question here again etc.

```